packages/applicant/src/pages/api/logout.page.tsx

```diff
@@ -1,12 +1,33 @@
-import { NextApiRequest, NextApiResponse } from 'next';
+import type { NextApiRequest, NextApiResponse } from 'next';
+import { getSessionIdFromCookies } from '../../../src/utils/session';
+import axios from 'axios';
 
-export default async function handler(
-  req: NextApiRequest,
-  res: NextApiResponse
-) {
-  res.redirect(
+const Logout = async (req: NextApiRequest, res: NextApiResponse) => {
+  const sessionCookie = getSessionIdFromCookies(req);
+  if (sessionCookie) await logoutAdmin(sessionCookie);
+  res.setHeader(
+    'Set-Cookie',
+    `session_id=deleted; Path=/; secure; HttpOnly; SameSite=Strict; expires=Thu, 01 Jan 2003 00:00:00 GMT`
```

Comment:
same as the other PR - do you not want to delete `user-service-token` every time and then delete `session_id` only when present?